### PR TITLE
add butons for download csv of ecosystems graphs

### PR DESCRIPTION
**Pull request by Trjegul84**

### CODE REVIEW

**Review by erikasv — CHANGES_REQUESTED** [review]
Please adjust styles with Cesar

**Comment by Trjegul84** [comment]
> Please adjust styles with Cesar

Cesar said too much buttons and leave just one for EE section.

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx [hunk 1]
```diff
@@ -87,19 +87,21 @@ class EcosystemDetails extends Component {
               <b>No hay información disponible de coberturas</b>
             )}
             {(coverageData && coverageData.length > 0) && (
-              <GraphLoader
-                graphType="SmallBarStackGraph"
-                data={coverageData}
-                units="ha"
-                colors={matchColor('coverage')}
-                onClickGraphHandler={(selected) => {
+
```
Comment (erikasv) [inline src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx:90]: delete this empty line

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx [hunk 2]
```diff
@@ -87,19 +87,21 @@ class EcosystemDetails extends Component {
...
+              colors={matchColor('coverage')}
+              onClickGraphHandler={(selected) => {
                   handlerClickOnGraph({
                     chartType: 'seCoverage',
                     chartSection: SEKey(SEValues.type),
                     selectedKey: selected,
                   });
                 }}
-              />
+            />
```
Comment (erikasv) [inline src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx:103]: Fix indentation

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx [hunk 3]
```diff
@@ -87,19 +87,21 @@ class EcosystemDetails extends Component {
...
+
```
Comment (erikasv) [inline src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx:104]: delete empty line

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx [hunk 4]
```diff
@@ -116,12 +118,14 @@ class EcosystemDetails extends Component {
...
-                colors={matchColor('pa', true)}
-              />
+
+            <GraphLoader
+              graphType="SmallBarStackGraph"
+              data={paData}
+              units="ha"
+              colors={matchColor('pa', true)}
+            />
+
```
Comment (erikasv) [inline src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx:128]: delete empty lines and fix indentation

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx (hunk 1 above)
Comment (Trjegul84) [reply src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx]: done!

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx (hunk 2 above)
Comment (Trjegul84) [reply src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx]: done!

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx (hunk 3 above)
Comment (Trjegul84) [reply src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx]: done!

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx (hunk 4 above)
Comment (Trjegul84) [reply src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx]: done!

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx [hunk 5]
```diff
@@ -87,19 +87,19 @@ class EcosystemDetails extends Component {
...
+              units="ha"
+              colors={matchColor('coverage')}
+              onClickGraphHandler={(selected) => {
+                handlerClickOnGraph({
+                  chartType: 'seCoverage',
+                  chartSection: SEKey(SEValues.type),
+                  selectedKey: selected,
+                });
+              }}
+            />
```
Comment (erikasv) [inline src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx:102]: internally indentation is ok, but all the block is missing one level (see that line 89is opening the parenthesis that closes on line 103)

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx [hunk 6]
```diff
@@ -116,12 +116,12 @@ class EcosystemDetails extends Component {
...
-                data={paData}
-                units="ha"
-                colors={matchColor('pa', true)}
-              />
+            <GraphLoader
+              graphType="SmallBarStackGraph"
+              data={paData}
+              units="ha"
+              colors={matchColor('pa', true)}
+            />
```
Comment (erikasv) [inline src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx:124]: same comment

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx (hunk 5 above)
Comment (Trjegul84) [reply src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx]: Ok, now is fixed!

File: src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx (hunk 6 above)
Comment (Trjegul84) [reply src/pages/search/drawer/strategicEcosystems/ecosystemsBox/EcosystemDetails.jsx]: Ok, now is fixed!

**Review by erikasv — APPROVED** [review]

**Pull request merged**